shared/utils/build-all-remediation-roles.py

```diff
@@ -79,6 +79,23 @@ def generate_role_for_input_content(input_content, benchmark_id, profile_id, tem
     return ret
 
 
+def add_minimum_ansible_version(ansible_src):
+    content = ""
+
+    for lines in ansible_src.split("\n"):
+        if lines == " - hosts: all":
```

Comment:
It splits a very big file to lines and then merges the lines together, line by line, using concatenation. That looks ineffective. Is it possible to just replace a substring by another substring in the playboook?